sites/public/src/pages/reset-password.tsx

```diff
@@ -38,13 +39,27 @@ const ResetPassword = () => {
     })
   }, [])
 
+  const handleQueryParams = (query: ParsedUrlQuery) => {
```

Comment:
This function is repeated twice and is only called once. I'd recommend just putting this inline, and you may be able to remove the typechecking to return undefined if you do so.